src/main/java/com/g4s8/ghman/env/EnvironmentVariables.java

```diff
@@ -95,7 +105,33 @@ public String githubClientSecret() throws IllegalStateException {
      */
     public String applicationHost() throws IllegalStateException {
         return this.envvars.computeIfAbsent(
-            this.APP_HOST,
+            EnvironmentVariables.APP_HOST,
+            this::missingKeyExceptionThrower
+        );
+    }
+
+    /**
+     * Retrieve Bot username.
+     * @return Bot username
+     * @throws IllegalStateException if bot username is missing from
+     *  system environment variables.
+     */
+    public String botUsername() throws IllegalStateException {
```

Comment:
@victornoel please add test method into `EnvironmentVariablesTest` for this new method behaviour or todo it